### PR TITLE
create_vm integration test

### PR DESCRIPTION
**Pull request by wen-quan**

### CODE REVIEW

File: integration/create_vm/create_vm_test.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		AfterEach(func() {
+			testhelpers.WaitForVirtualGuestToHaveNoActiveTransactions(int(vmId))
+			testhelpers.DeleteVirtualGuest(int(vmId))
+		})
+
+		It("returns true because valid parameters", func() {
+			jsonPayload, err := testhelperscpi.GenerateCpiJsonPayload("create_vm", rootTemplatePath, replacementMap)
+			Expect(err).ToNot(HaveOccurred())
+
+			//The outputBytes always be a JSON string.
```
Comment (maximilien) [inline integration/create_vm/create_vm_test.go]: Why do you leave this comment? Was it to help in develop the test and you forgot to remove or is this information...

From the expectations I feel you should simply remove this.

**Comment by maximilien** [comment]
Hi, @wen-quan,

Looks good to me, except for the comment you left. So if you remove it then I will merge.

The only thing I'd like to monitor is impact on integration tests time after adding this to pipeline. So @posijon once you see this merged let's keep an eye on whether integration takes considerable amount of time more.

Best,

max

**Comment by wen-quan** [comment]
Dear max, 
Great thanks for your suggestion. 
I will remove these comments.

Best,

wenquan

**Comment by maximilien** [comment]
Ok cool. Merging. /cc @posijon so we can monitor impact on pipeline.

Cheers :beers:

**Pull request merged**